frontend/src/pages/PageTestbed.vue

```diff
@@ -251,6 +260,38 @@ function sendMessage() {
   );
 }
 
+/** Same as above, but for multi-compare phenogrid */
+function postPhenogridMulti() {
+  const iframe =
+    document.querySelector<HTMLIFrameElement>("[name=pheno-multi]");
+  if (!iframe) return;
+  iframe.contentWindow?.postMessage(
```

Comment:
Even if it's just hardcoded here, we should swap in the values from the multi compare request in the fixtures